pkg/volume/glusterfs/glusterfs.go

```diff
@@ -721,6 +721,7 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		glusterTypeAnn:                             "file",
 		"Description":                              glusterDescAnn,
 		v1.MountOptionAnnotation:                   "auto_unmount",
+		"VolID":                                    volID,
```

Comment:
This should be some `gluster.kubernetes.io/volume-id` or something like that.

Reply:
Changed it to `gluster.kubernetes.io/heketi-volume-id` which is more conveying.